xcp_d/utils/hcp2fmriprep.py

```diff
@@ -113,30 +143,61 @@ def hcpfmriprepx(hcp_dir, out_dir, subid):
 
     for i, j in zip(hcpfiles, newanatfiles):
         copyfileobj_example(i, j)
-
+    print("finished converting anat files")
     # get the task files
-    taskx = glob.glob(hcp_dir + "/" + subid + "/MNINonLinear/Results/*")
+
+    taskx = glob.glob(hcp_dir + "/Results/*")
     tasklistx = []
+    print(hcp_dir)
     for j in taskx:
         if j.endswith("RL") or j.endswith("LR"):
             tasklistx.append(j)
 
-    csf_mask = pkgrf("xcp_d", "masks/csf.nii.gz")
-    wm_mask = pkgrf("xcp_d", "masks/wm.nii.gz")
-
+    csf_mask = pkgrf("xcp_d", "/data/masks/csf.nii.gz")
+    wm_mask = pkgrf("xcp_d", "/data/masks/wm.nii.gz")
```

Comment:
We should rename these to be more informative. I'm guessing they're in a specific MNI space+resolution.

Reply:
will make a note